client/cli/src/config.rs

```diff
@@ -278,13 +283,12 @@ pub trait CliConfiguration: Sized {
 		Ok(Default::default())
 	}
 
-	/// Returns `Ok(true)` if offchain worker should be used
+	/// Returns a offchain worker config wrapped in `Ok(_)`
 	///
-	/// By default this is `false`.
-	fn offchain_worker(&self, _role: &Role) -> Result<bool> {
+	/// By default offchain workers are disabled.
+	fn offchain_worker_config(&self) -> Result<OffchainWorkerConfig> {
```

Comment:
Since offchain_worker_params is available in this trait, you need to change the default implementation of offchain_worker_config to make it try to use that instead.

In other words: you can move the function offchain_worker_config from runcmd.rs to config.rs. 

This will allow another command to use OffchainWorkerParams just by defining offchain_worker_params. Does that make sense?

Reply:
Thanks!